src/test/resources/testPatterns.conf

```diff
@@ -0,0 +1,12 @@
+ExtractionPatterns = {
+     patterns = [
+         {
+             name = "result-percent",
+             pattern = "CD {%|percent|per cent|pct}"
+           },
+         {
+             name = "treatements",
+             pattern = "{were given|treated with|received|receiving} {CD|NN|JJ|IN}+"
+           }
+     ]
+   }
```

Comment:
Indentation here is a little unconventional.  This is usually how our configs are formatted:
```
ExtractionPatterns = {
  patterns = [
    {
      name = "result-percent",
      pattern = "CD {%|percent|per cent|pct}"
    },
    {
      name = "treatements",
      pattern = "{were given|treated with|received|receiving} {CD|NN|JJ|IN}+"
    }
  ]
}
```